### DISCUSSION TITLE
Jupyter Widget Missing Extension?

### DISCUSSION THREAD

**Discussion by Daviology38** [Unidata/pyaos-ams-2021#130] [Q&A]
Good afternoon!

I am currently working through the Widgets Training Notebook and cannot get the widget to display in the notebook on JupyterLab. I receive this instead "IntSlider(value=5, description='n', max=10, min=1)", which googling stack overflow tells me that it is a missing extension. 

Is this the case?

Thanks,
Dave

**Comment by dcamron** [comment]
Hi Dave! Thanks for the question. Right now, the quickest way to fix this is to create and execute a new cell in your notebook:

`!jupyter lab build`

(don't forget the exclamation point!) This will rebuild jupyter lab and its extensions (one of which is the jupyterlab-widgets extensions), and it might take a few minutes, so be patient! Afterwords, the widgets should display correctly. If not, follow up again and let us know here. We might be able to find a way to fix this for everyone at once while we're at it.

**Reply by Daviology38** [reply]
Thanks for the reply!

I tried both the build solution above and installing the extension manually and neither of them worked. 

Looking into it further, an "Exception opening new comm" appears in my browser javascript console when I run any of the widget cells. I have tried updating the nodejs on the jupyterlab, to no avail, so I'm not entirely sure where the error lies.

**Reply by haileyajohnson** [reply]
Hi Dave! Sorry that didn't work for you. We're working on a solution, but in the meantime, if you want to try running it locally, there are instructions for that [here](https://github.com/Unidata/pyaos-ams-2021/discussions/126). Running locally has been working for everyone so far.

We'll let you know if we figure out anything else!
